packages/alfa-dom/src/node/element.ts

```diff
@@ -49,6 +52,7 @@ export class Element<N extends string = string>
   private readonly _name: N;
   private readonly _attributes: Map<string, Attribute>;
   private readonly _style: Option<Block>;
+  private readonly _rectangle: Option<Rectangle>;
```

Comment:
**Suggestion:** I think this should be called "bounding box" or something like that (i.e. a semantic name, rather than a technical one).
OTOH, this is not exactly any of the CSS boxes 🙈 https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect
Maybe `box` is generic enough?